docsite/rst/intro_inventory.rst

```diff
@@ -261,6 +260,14 @@ Remote host environment parameters::
       Works for anything such as ruby or perl and works just like ansible_python_interpreter.
       This replaces shebang of modules which will run on that host.
 
+.. versionadded:: 2.1
+
+::
+
+    ansible_executable
+      This sets the shell the ansible controller will use on the target machine, overrides ``executable`` in ``ansible.cfg`` which defaults to '/bin/sh'.
+      You should really only change it if is not possible to use '/bin/sh' (i.e. it is not in the list of allowed shells for your users).
+
```

Comment:
I don't think allowed shells matters because we're invoking it as a command, not as a login shell.  So this probably should be if /bin/sh is not available or not present on the machine.